src/components/LobbyErrorDialog.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <BaseDialog
+    id="ErrorDialog"
+    :model-value="showLobbyError"
+    title="Oops!"
+    :opacity="1"
+    data-cy="create-game-dialog"
+  >
+    <template #body>
+      <div>
+        <h4>Cannot Join Lobby {{ errorMessage }}</h4>
+      </div>
+    </template>
+    <template #actions>
+      <v-btn
+        data-cy="how-it-works-okay"
```

Comment:
This data-cy looks inaccurate
```suggestion
        data-cy="lobby-error-go-home"
```